File: examples-sdk/swift/202-llm-multi-model-routing/main.swift
// 202-llm-multi-model-routing — Swift SDK equivalent
let p = VilPipeline(name: "MultiModelPipeline_GPT4", port: 8080)
p.route(from: "sink.trigger_out", to: "source_gpt4.trigger_in", mode: "LoanWrite")
p.route(from: "source_gpt4.response_data_out", to: "sink.response_data_in", mode: "LoanWrite")
p.route(from: "source_gpt4.response_ctrl_out", to: "sink.response_ctrl_in", mode: "Copy")
p.compile()

File: examples-sdk/go/202-llm-multi-model-routing/main.go
// 202-llm-multi-model-routing — Go SDK equivalent
// Compile: vil compile --from go --input 202-llm-multi-model-routing/main.go --release
package main

import vil "github.com/OceanOS-id/vil-go"

func main() {
	p := vil.NewPipeline("MultiModelPipeline_GPT4", 8080)

	p.Route("sink.trigger_out", "source_gpt4.trigger_in", "LoanWrite")
	p.Route("source_gpt4.response_data_out", "sink.response_data_in", "LoanWrite")
	p.Route("source_gpt4.response_ctrl_out", "sink.response_ctrl_in", "Copy")

	p.Compile()
}
